### PR TITLE
Handle decrement method.

### LINKED ISSUE #61: Missing decrement method in events
The `decrement` method is not managed by the bundle. 

Consider the following configuration:

```yaml
m6_statsd:
  clients:
    default:
      events:
        order.accepted:
          decrement: "orders.waiting"
```

It causes the following error: 
```
Unrecognized option "decrement" under "m6_statsd.clients.default.events.order.accepted"
```

I'll send a pull request shortly.

### PR DESCRIPTION
**Pull request by alexsegura**
Fixes #61 

Still need to add tests.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -27,12 +27,12 @@ This bundle is tested with [atoum](https://github.com/atoum/atoum).
...
 To launch tests, you need to install dependencies in dev mode
 ```bash
-composer install --dev
+composer install
 ```
 
 And you can now launch tests
 ```bash
-./vendor/bin/atoum
+./bin/atoum
```
Comment (Oliboy50) [inline CONTRIBUTING.md]: you could also remove the `./` to only keep `bin/atoum`

Comment (alexsegura) [reply CONTRIBUTING.md]: Done.

**Comment by alexsegura** [comment]
Done.

@omansour @Oliboy50 do you think you can merge this and create a new release? Thanks 😬

**Comment by omansour** [comment]
dammit ! with tests not running for the 0508049  commit ...

**Comment by omansour** [comment]
v2.15.0

**Pull request merged**